upgrade-plugins/src/main/java/org/exoplatform/ecms/upgrade/templates/NodeTypeTemplateUpgradePlugin.java

```diff
@@ -91,11 +119,31 @@ public void processUpgrade(String oldVersion, String newVersion) {
       // remove all old node type nodes
       for (Node removedNode : removedNodes) {
         try {
-          removedNode.remove();
+          String removedTemplateName = removedNode.getName();
+          //if Template had not been edited before, remove it
+          if(!modifiedTemplateLog.contains(removedTemplateName)){
+            if (log.isInfoEnabled()) {
+              log.info("old template {} will be removed, and new template will be imported", removedTemplateName);
```

Comment:
String removedTemplateName = removedNode.getName();
so removedTemplateName is NodeName